ocrd_models/ocrd_models/ocrd_exif.py

```diff
@@ -39,7 +39,8 @@ def __init__(self, img):
         if img.format in ('TIFF', 'PNG') and 'dpi' in img.info:
             self.xResolution = int(img.info['dpi'][0])
             self.yResolution = int(img.info['dpi'][1])
-            self.resolutionUnit = 'cm' if img.tag.get(296) == 3 else 'inches'
+            if img.format == 'TIFF':
+                self.resolutionUnit = 'cm' if img.tag.get(296) == 3 else 'inches'
```

Comment:
```suggestion
                self.resolutionUnit = 'cm' if img.tag.get(296) == 3 else 'inches'
            else:    
                self.resolutionUnit = 'inches'
```

Reply:
Also, you still have to set `self.resolutionUnit = 'inches'` down below in the other `PNG` case. (Sorry, cannot make suggestions outside of hunk context.

Reply:
Another (minor) thing: the docstring wrongly describes that attribute as `resolutionUnits` (plural)